### PR TITLE
Release v0.13.0

### PR DESCRIPTION
**Pull request by fredcarle**
Release branch as of 470343681f4906fa1b8605d108d3cf34e8f18e56

Release tasks completed:

- [x] ran go mod tidy
- [x] regenerated in-repo docs (CLI and manpages)
- [x] bump BSL license parameters (Version and Date)
- [x] update changelog
- [x] Made sure that the Playground can build

Only need to review the last commit :)

### CODE REVIEW

File: licenses/BSL.txt [hunk 1]
```diff
@@ -7,7 +7,7 @@ Parameters
 Licensor:                   Democratized Data (D2) Foundation
 
 
-Licensed Work:              DefraDB v0.11.0
+Licensed Work:              DefraDB v0.13.0
```
Comment (AndrewSisley) [inline licenses/BSL.txt:10]: :)

**Review by AndrewSisley — APPROVED** [review]
LGTM, thanks Fred!

File: licenses/BSL.txt (hunk 1 above)
Comment (fredcarle) [reply licenses/BSL.txt:10]: I know... That and the changelog file copy that got forgotten 🤦‍♂️

**Review by nasdf — APPROVED** [review]
LGTM

**Review by shahzadlone — APPROVED** [review]

**Pull request merged**